Automattic/wp-calypso
# Convert Event Countdown and Timeline blocks to use SCSS build

#### Changes proposed in this Pull Request

Moves the CSS for the Event Countdown and Timeline blocks to use the build setup with FSE plugin. Also simplified the PHP a little to match other plugins.

#### Testing instructions

* No feature or functionality changes
* Confirm after build the blocks show as expected in editor and front-end
